### DISCUSSION TITLE
Can a modifier apply to an element instead of a block?

### DISCUSSION THREAD

**Discussion by pupeno** [getbem/getbem.github.io#254] [Q&A]
For example, if you have an a list, with list items, and one of those items are active, should it be specified like this:

```css

.list { }
.list__item { }
.list__item--active { }
```

How would one differentiate from a list that is active instead of a list item that is active?

**Comment by madroneropaulo** [comment]
`.list__item--active` is already an element modifier.

If you want a modifier for your list, just add it to the block:

`.list--active__item--active`.

This way you have a selector for an item--active  for a list--active
